### DISCUSSION TITLE
add model settings shortcut for training

### DISCUSSION THREAD

**Discussion by alinelena** [ACEsuit/mace#734] [Ideas]
at the moment when you run mace_run_train you need to specify the architecture details.

would be nice to have --small --medium --large that automatically set all the arch details to defaults uses in mace_mp.

**Comment by gabor1** [comment]
This is a difficult choice. if we do that, then people will never experiment with the options, and when we decide to change them one day, everyone will complain about the incompatibility. We don't really know enough yet about the accuracy tradeoffs between channel number and the other hyper parameters (ell, L, body order etc)

**Reply by alinelena** [reply]
I agree with the corner case, but having some shortcuts (aligned with the mace_mp paper) may help newcomers to start things up...

**Reply by ilyes319** [reply]
I think new comers should finetune and then they have it already.
